From be4b9f35cd645856b7bef3b1ba0193b6a1eb35ae Mon Sep 17 00:00:00 2001
From: chenxiaolong <chenxiaolong@loongson.cn>
Date: Fri, 5 Jan 2024 11:43:27 +0800
Subject: [PATCH 212/303] LoongArch: testsuite:Delete the default run behavior
 in pr60510.f.

When binutils does not support vector instruction sets, the test program fails
because it does not recognize vectorization at the assembly stage. Therefore,
the default run behavior of the program is deleted, so that the behavior of
the program depends on whether the software supports vectorization.

gcc/testsuite/ChangeLog:

	* gfortran.dg/vect/pr60510.f: Delete the default behavior of the
	program.
---
 gcc/testsuite/gfortran.dg/vect/pr60510.f | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gcc/testsuite/gfortran.dg/vect/pr60510.f b/src/gcc/testsuite/gfortran.dg/vect/pr60510.f
index ecd50dd55..c1e11b27d 100644
--- a/src/gcc/testsuite/gfortran.dg/vect/pr60510.f
+++ b/src/gcc/testsuite/gfortran.dg/vect/pr60510.f
@@ -1,4 +1,3 @@
-! { dg-do run }
 ! { dg-require-effective-target vect_double }
 ! { dg-require-effective-target vect_intdouble_cvt }
 ! { dg-additional-options "-fno-inline -ffast-math" }
-- 
2.33.0

